### PR TITLE
Add VD2's removal policy for ToS2 (+ caching)

### PR DESCRIPTION
**Pull request by MaelRL**
## Summary of Change

Adaptation policies for triangulations on the sphere

## Release Management

* Affected package(s): `Triangulation_on_sphere_2` `Voronoi_diagram_2`
* Issue(s) solved (if any): -
* Feature/Small Feature (if any): -
* License and copyright ownership: no change

### CODE REVIEW

File: Voronoi_diagram_2/include/CGAL/Delaunay_triangulation_on_sphere_adaptation_policies_2.h [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+//
+// Author(s)     : Menelaos Karavelas <mkaravel@iacm.forth.gr>
+//                 Mael Rouxel-Labbé
+
+#ifndef CGAL_DELAUNAY_TRIANGULATION_ON_SPHERE_ADAPTATION_POLICIES_2_H
+#define CGAL_DELAUNAY_TRIANGULATION_ON_SPHERE_ADAPTATION_POLICIES_2_H 1
```
Comment (sloriot) [inline Voronoi_diagram_2/include/CGAL/Delaunay_triangulation_on_sphere_adaptation_policies_2.h:15]: do we have any other file with the `1`?

Comment (lrineau) [reply Voronoi_diagram_2/include/CGAL/Delaunay_triangulation_on_sphere_adaptation_policies_2.h:15]: Quite a lot:
```shellsession
% ack '# *define.*_H *1' */include -l | wc -l      
227
```

Comment (sloriot) [reply Voronoi_diagram_2/include/CGAL/Delaunay_triangulation_on_sphere_adaptation_policies_2.h:15]: less than 7% of the include files. I do not really care, it just caught my eye and I directly thought of a special case when writing scripts :)

**Comment by maxGimeno** [comment]
Error : https://cgal.geometryfactory.com/CGAL/testsuite/results-5.3-Ic-163.shtml#Voronoi_diagram_2

**Comment by maxGimeno** [comment]
warning: https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-5.4-Ic-13/Voronoi_diagram_2/TestReport_lrineau_Ubuntu-latest-GCC6-CXX1z.gz

**Comment by maxGimeno** [comment]
tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.4-Ic-15.shtml

**Pull request merged**